### PR TITLE
beginning of project

### PR DESCRIPTION
**Pull request by katiemacaulay**
## Checkpoint Rubric
This is the rubric that your instructor will use to grade your checkpoints. Please do not edit. 

**Checkpoint 1**
- [ ] All tests passed: 40 points
- [ ] Proper use of documentation (commenting on code): 15 points
- [ ] Properly indented code: 15 points
- [ ] Demonstrated effective use of JavaScript: 30 points

**Checkpoint 2**
- [ ] The application works as it should: 40 points
- [ ] Proper use of documentation (commenting on code): 15 points
- [ ] Properly indented code: 15 points
- [ ] Demonstrated effective use of JavaScript and the DOM API: 30 points

**Checkpoint 3**
- [ ] Use of React: 25 points
- [ ] Accesses an API: 25 points
- [ ] Proper use of documentation (commenting on code): 25 points
- [ ] The application functions as it should: 25 points

### CODE REVIEW

File: 01week/datatypes.js [hunk 1]
```diff
@@ -0,0 +1,80 @@
+'use strict'
+
+// Write a JavaScript program to display the current day and time.
+
+const currentDate = new Date();
+const currentDateAndTime = (currentDate.getMonth() + 1) + '/' + currentDate.getDate() + '/' +
```
Comment (reneemeyer) [inline 01week/datatypes.js]: this needs to be a function that returns today's date

File: 01week/datatypes.js [hunk 2]
```diff
@@ -0,0 +1,80 @@
...
+
+const currentDate = new Date();
+const currentDateAndTime = (currentDate.getMonth() + 1) + '/' + currentDate.getDate() + '/' + 
+currentDate.getFullYear();
+
+console.log(currentDateAndTime);
+
+// Write a JavaScript program to convert a number to a string.
+
+const convertToString = (num1) => num1.toString();
```
Comment (reneemeyer) [inline 01week/datatypes.js:14]: this needs to be a function that takes in any number and returns the number converted as a string

File: 01week/datatypes.js [hunk 3]
```diff
@@ -0,0 +1,80 @@
...
+
+console.log(typeof convertToString(20))
+
+// Write a JavaScript program to convert a string to the number.
+
+const convertToNumber = (stringValue1) => Number(stringValue1);
```
Comment (reneemeyer) [inline 01week/datatypes.js:20]: this needs to be a function that takes in any string and converts it to a number

**Review by reneemeyer — CHANGES_REQUESTED** [review]
Good start, please switch back to this branch, address the comments below, and push to the branch.

File: 01week/datatypes.js (hunk 3 above)
Comment (katiemacaulay) [reply 01week/datatypes.js:20]: Hi Renee, just pushed up an updated version.  Thank you.

**Pull request merged**